src/components/CommentTeaser/CommentTeaserHeader.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { css } from 'glamor'
+import colors from '../../theme/colors'
+import { sansSerifMedium16, sansSerifRegular14 } from '../Typography/styles'
+
+const styles = {
+  root: css({
+    display: 'flex',
+    flexDirection: 'column'
+  }),
+  title: css({
+    ...sansSerifMedium16,
+    color: colors.text,
+    whiteSpace: 'nowrap',
+    overflow: 'hidden',
+    textOverflow: 'ellipsis'
+  }),
+  subtitle: css({
+    ...sansSerifRegular14,
+    lineHeight: 1.1,
```

Comment:
Why 1.1?

Reply:
Because of this:
<img width="249" alt="screen shot 2017-11-01 at 17 24 24" src="https://user-images.githubusercontent.com/23520051/32285132-ac06f7f6-bf29-11e7-8028-8601c83241c6.png">

The descenders (like the "g"s here) are cut off at a line-height of 1.0, because I use an "overflow: hidden" here for ellipsizing the potentially long subtitle. A line-height of 1.1 fixes that.